DokanStore/Networking/Package.swift

```diff
@@ -0,0 +1,34 @@
+// swift-tools-version: 5.6
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Networking",
+    platforms: [
+         .macOS(.v10_12),
+         .iOS(.v10),
+     ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Networking",
+            targets: ["Networking"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/Moya/Moya.git", .upToNextMajor(from: "15.0.0"))
```

Comment:
We dropped Moya and now using Alamofire instead

Reply:
I told Eng.Ahmed I want to use Moya

Reply:
Yeah, sorry I thought this would be merged to the main branch